Sources/Components/RealestateAgencyContent/Subviews/RealestateAgencyContentItemView.swift

```diff
@@ -74,20 +74,25 @@ class RealestateAgencyContentItemView: UIView {
 
     private func configure(
         with article: RealestateAgencyContentViewModel.ArticleItem,
-        colors: RealestateAgencyContentViewModel.Colors,
+        styling: RealestateAgencyContentViewModel.Styling,
         remoteImageViewDataSource: RemoteImageViewDataSource
     ) {
-        titleLabel.textColor = colors.main.text
-        bodyLabel.textColor = colors.main.text
+        titleLabel.textColor = styling.textColor
+        bodyLabel.textColor = styling.textColor
 
         titleLabel.text = article.title
         bodyLabel.text = article.body
 
         imageView.dataSource = remoteImageViewDataSource
         imageView.loadImage(for: article.imageUrl, imageWidth: .zero, loadingColor: .sardine)
 
-        let actionButton = Button.create(for: article, textColor: colors.actionButton.text, backgroundColor: colors.actionButton.background)
-        buttonStackView.addArrangedSubviews([actionButton, UIView(withAutoLayout: true)])
+        let actionButton = Button.create(for: article, styling: styling)
+        buttonStackView.addArrangedSubview(actionButton)
+
+        if article.buttonKind == .normal {
+            buttonStackView.addArrangedSubview(UIView(withAutoLayout: true))
```

Comment:
is it a spacing view?

Reply:
Yep, only the highlighted buttons should use the whole width